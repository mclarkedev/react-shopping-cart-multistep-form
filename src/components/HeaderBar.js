import React from 'react';
import Hamburger from '../img/menu.svg';
import Search from '../img/search.svg';
import Cart from '../img/cart.svg';
import Header from '../styles/elements/header';

const HeaderBar = () => {
    return (
        <Header>
            <div className="brand">
                <div className="hamburger" aria-label="Open mobile navigation menu">
                    <img src={Hamburger} alt="" />
                </div>
                <p>MyStore</p>
            </div>
            <nav>
                <ul>
                    <li>Women</li>
                    <li>Men</li>
                    <li>Kids</li>
                    <li>Sale</li>
                </ul>
            </nav>
            <div className="nav-icons">
                <div aria-label="Click to open search">
                    <img src={Search} alt="Search" />
                </div>
                <div aria-label="Click to go to cart">
                    <img src={Cart} alt="Shopping cart" />
                </div>
            </div>
        </Header>
    )
}

export default HeaderBar;