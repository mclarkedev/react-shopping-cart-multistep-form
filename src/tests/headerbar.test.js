import React from 'react';
import { shallow } from 'enzyme';
import HeaderBar from '../components/HeaderBar';

describe('Header Component', () => {
    it('should render without errors', () => {
        const wrapper = shallow(<HeaderBar />);
        expect(wrapper).toMatchSnapshot();
    });

    it('should render children', () => {
        const component = shallow(<HeaderBar />);
        const wrapper = component.find('nav');
        expect(wrapper.length).toBe(1);
    });
})