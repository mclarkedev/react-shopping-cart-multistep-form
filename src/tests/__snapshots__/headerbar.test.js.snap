// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render without errors 1`] = `ShallowWrapper {}`;
